src/components/urls/helpers.tsx

```diff
@@ -60,3 +60,8 @@ export const Copy = ({ text, message, children }: CopyProps) => <BlackLink
     showInfoMessage(message)
   }}
 >{children}</BlackLink>
+
+export const fullPath = (relative: string) => { 
+  const pathname = relative.startsWith('/') ? relative : '/' + relative
+  return appBaseUrl + pathname
```

Comment:
This logic looks bad. It should check that the relative URL is not resolved to a full URL already. Currently, I can easily pass a full URL to this func and it will prepend it with appBaseUrl once more – that is wrong.